api/api/templates/admin/api/media_report/change_form.html

```diff
@@ -0,0 +1,159 @@
+{% extends "admin/change_form.html" %}
+{% load static %}
+
+{% block extrahead %}
+{{ block.super }}
+<style>
+  audio, .img-container {
+    margin-top: 1rem;
+    margin-bottom: 1rem;
+  }
+  .img-container {
+    display: grid;
+    justify-content: center;
+    align-items: center;
+  }
+  .img-container img, .img-container p {
+    grid-column: 1 / -1;
+    grid-row: 1 / -1;
+  }
+  .img-container p {
+    border-radius: 4px;
+    padding: 10px;
+    z-index:1;
+    pointer-events: none;
+    color: black;
+    background: rgba(255, 255, 255, 0.5);
+    text-align: center;
+    font-weight: bold;
+  }
+  .blur-image {
+    transition: filter 0.3s;
+    filter: blur(60px);
+  }
+  .no-blur {
+    filter: none;
+  }
+  .no-blur ~ p {
+    display: none;
+  }
+</style>
+{% endblock %}
+
+{% block after_field_sets %}
+{{ block.super }}
+
+{% if not add %}
+<fieldset class="module aligned">
+  <h2>Media details</h2>
+  {% if media_type == 'image' %}
+    <div class="img-container">
+      <img src="{% url media_type|add:'-thumb' identifier=identifier %}" alt="Media Image" class="blur-image" height="300"
+           onclick="toggleBlur(this)"
+           onerror="this.onerror=null;this.src='{{ url }}';"
+           style="cursor: pointer;">
+      <p>Show content</p>
+    </div>
+
+  {% elif media_type == 'audio' %}
+    <audio controls>
+      <source src="{{ url }}" type="audio/mpeg">
```

Comment:
Is audio/mpeg safe? What if it's any of the other codecs supported ([webm, for example](https://api.openverse.engineering/v1/audio/?extension=webm))? I think the browser can be left to deduce the correct type on its own :thinking:. Is `type` a required attribute for audio `source` element?

```suggestion
      <source src="{{ url }}" type="audio/mpeg">
```

Reply:
Removed, doesn't cause any warnings without it.